src/client/protocolgameparse.cpp

```diff
@@ -1587,7 +1595,10 @@ void ProtocolGame::parsePlayerStats(const InputMessagePtr& msg)
         m_localPlayer->setBaseMagicLevel(baseMagicLevel);
     }
 
-    const double soul = msg->getU8();
+    double soul = 0;
+    if (g_game.getFeature(Otc::GameSoulStat))
```

Comment:
rename to GameSoul

Reply:
Done.

Reply:
This exact line @nekiro mentioned is not corrected in the commit 68d3189943be6da18bef730a4898acbb040d7cc4

Reply:
Awkward. Commited a fix (and actually compiled it this time)